Server/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Server
+{
+    public class WeatherForecast
```

Comment:
delete this file not needed in as per out task

Reply:
deleted the weatherforecast.cs file